server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java

```diff
@@ -176,6 +176,13 @@ public PhysicalPlan transformToPhysicalPlan(Operator operator) throws QueryProce
               "For Insert command, cannot specified more than one seriesPath: " + paths);
         }
 
+        if (insert.getMeasurementList().length != insert.getValueList().length) {
+          throw new ArrayIndexOutOfBoundsException(
```

Comment:
You could throw an SQLParserException.

Reply:
Thank you for your opinion. I will revise it